software/deploy/deploy.go

```diff
@@ -427,3 +433,27 @@ func DagsOnlyDeploy(houstonClient houston.ClientInterface, appConfig *houston.Ap
 	}
 	return fileutil.UploadFile(&uploadFileArgs)
 }
+
+func UpdateDeploymentImage(houstonClient houston.ClientInterface, deploymentID, wsID, runtimeVersion, imageName string) error {
+	if runtimeVersion == "" {
+		return ErrNoRuntimeVersionPassed
+	}
+	if imageName == "" {
+		return ErrNoImageNamePassed
+	}
+	deploymentIDForCurrentCmd, _, err := getDeploymentIDForCurrentCommandVar(houstonClient, wsID, deploymentID, deploymentID == "")
+	if err != nil {
+		return err
+	}
+	deploymentID = deploymentIDForCurrentCmd
+	if deploymentID == "" {
+		return errInvalidDeploymentID
+	}
```

Comment:
```suggestion
	deploymentID, _, err := getDeploymentIDForCurrentCommandVar(houstonClient, wsID, deploymentID, deploymentID == "")
	if err != nil {
		return err
	}
	if deploymentID == "" {
		return errInvalidDeploymentID
	}
```
nit: simplifying the logic